src/PrestaShopBundle/Resources/views/Admin/base.html.twig

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="{{ iso }}">
+<head>
+  {# ToDo: check for dynamic header #}
+  {% include ['@PrestaShop/Admin/header.html.twig'] %}
+  {% block stylesheets %}{% endblock %}
+  {% block extra_stylesheets %}{% endblock %}
+</head>
+
+<body
+  class="lang-{{ iso }}{% if lang_is_rtl %} lang-rtl{% endif %} {{ controller_name|escape|lower }}{% if collapse_menu %} page-sidebar-closed{% endif %}"
+  data-base-url="{{ js_router_metadata.base_url }}"
+  data-token="{{ js_router_metadata.token }}"
+>
+
+{% if display_header %}
+  <header id="header" class="d-print-none">
+
+    <nav id="header_infos" class="main-header">
+      <button class="btn btn-primary-reverse onclick btn-lg unbind ajax-spinner"></button>
+
+      {# Logo #}
+      <i class="material-icons js-mobile-menu">menu</i>
+      <a id="header_logo" class="logo float-left" href="{{ default_tab_link|escape }}"></a>
+      <span id="shop_version">{{ ps_version }}</span>
+
+      <div class="component" id="quick-access-container">
+        {% include ['@PrestaShop/Admin/Components/quick_access.html.twig'] %}
+      </div>
+      <div class="component" id="header-search-container">
+        {% include ['@PrestaShop/Admin/Components/search_form.html.twig'] with {'show_clear_btn': false} %}
+      </div>
+
+      {% if debug_mode %}
+        <div class="component hide-mobile-sm" id="header-debug-mode-container">
+          <a class="link shop-state"
+             id="debug-mode"
+             data-toggle="pstooltip"
+             data-placement="bottom"
+             data-html="true"
+             title="<p class='text-left'><strong>{{ 'Your shop is in debug mode.'|trans({}, 'Admin.Navigation.Notification') }}</strong></p><p class='text-left'>{{ 'All the PHP errors and messages are displayed. When you no longer need it, [1]turn off[/1] this mode.'|trans({'[1]': '<strong>', '[/1]': '</strong>'}, 'Admin.Navigation.Notification') }}"
+             href="{{ getAdminLink('AdminPerformance')|escape }}"
+          >
+            <i class="material-icons">bug_report</i>
+            <span>{{ 'Debug mode'|trans({}, 'Admin.Navigation.Header') }}</span>
+          </a>
+        </div>
+      {% endif %}
+
+      {% if maintenance_mode %}
+        <div class="component hide-mobile-sm" id="header-maintenance-mode-container">
+          <a class="link shop-state"
+             id="maintenance-mode"
+             data-toggle="pstooltip"
+             data-placement="bottom"
+             data-html="true"
+             title="<p class='text-left'><strong>{{ 'Your shop is in maintenance.'|trans({}, 'Admin.Navigation.Notification') }}</strong></p><p class='text-left'>{{ 'Your visitors and customers cannot access your shop while in maintenance mode.%sTo manage the maintenance settings, go to Shop Parameters > Maintenance tab.'|trans({'%s': '<br>'}, 'Admin.Navigation.Notification') }}</p>" href="{{ getAdminLink('AdminMaintenance')|escape }}"
```

Comment:
Is there a reason for the %s to be placed here? :)

Reply:
Yep, this is where a line break occurs ;)